packages/progress-bar/src/ProgressBar/ProgressBar.tsx

```diff
@@ -72,8 +85,10 @@ export function ProgressBar(props: ProgressBarProps) {
 
       <div
         role={role}
-        id={progressBarId}
-        aria-label={progressBarId}
+        id={barId}
+        aria-labelledby={label ? labelId : undefined}
+        aria-label={!label ? role : undefined}
```

Comment:
We should probably require an `aria-label` prop if a label node isn't passed in.
See [`AriaLabelProps`](/packages/a11y/src/AriaLabelProps.ts)